### ISSUE TITLE
Thanks for this!

### ISSUE THREAD

**Issue by harmenjanssen** [igneus/php-functional-util-comparison#1]
Not a real issue of course: I'm the principal author of **Garp\Functional** and stumbled upon this repository. 

Super helpful, thanks for taking the time creating it! 

Also: your con as listed below **Garp\Functional** is spot on and you inspired me to fix it immediately. I'm going to take care of it. 
I actually didn't know how, but your comments on NSPL pointed me in the right direction.

So thanks again! 👌

**Comment by igneus** [comment]
Hi, thanks for letting me know. Feel free to ping me or open a PR once the new code is released.

NSPL is the first library of this kind I stumbled upon and started using a year or two ago, so it's the one I know best. (And it's quite possible that I miss important features of some other libraries due to my "NSPL mindset".)

**Comment by harmenjanssen** [comment]
Hi @igneus, I just tagged a new release containing function name constants! See 3.1.1 in the changelog: https://github.com/grrr-amsterdam/garp-functional/blob/master/CHANGELOG.md

By the way, what might be a nice feature to consider is the fact that Garp\Functional contains [Typeclasses](https://github.com/grrr-amsterdam/garp-functional/blob/master/docs/index.md#typeclasses), like Haskell has, for instance.
But the going is slow, and I've only implemented two in `master`. Semigroup and Monoid are coming though, with Functor following after that. 

Well, thanks again for pointing me in the direction of those function name constants. 🙂  
I'll close this issue.
